app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -480,21 +504,36 @@ class OnboardingFragmentTest {
   }
 
   @Test
-  // TODO(#973): Fix OnboardingFragmentTest
-  @Ignore
   fun testOnboardingFragment_clickOnSkip_changeOrientation_titleIsCorrect() {
     launch(OnboardingActivity::class.java).use {
       onView(withId(R.id.skip_text_view)).perform(click())
+      testCoroutineDispatchers.runCurrent()
       onView(isRoot()).perform(orientationLandscape())
       onView(
         allOf(
           withId(R.id.slide_title_text_view),
-          isDisplayed()
+          isCompletelyDisplayed()
         )
       ).check(matches(withText(R.string.onboarding_slide_3_title)))
     }
   }
 
+  private fun scrollToPage(position: Int): ViewAction {
```

Comment:
Rename this to `scrollToPosition`

Also, if you have referenced this code from somewhere please add that reference just above the functions in comments.

Also where-ever you are using this function use it like this:
`scrollToPosition(position = 1)`

Reply:
Done

Reply:
@rt4914 Actually, I didn't find any reference for it, so I tried creating it myself and it worked.

Reply:
That really nice.